### ISSUE TITLE
Let user introduce the final amount when buy from Coinbase is requested

### ISSUE THREAD

**Issue by miguelmorales85** [bitpay/copay#6971]
There is a difference when BTC is bought from Coinbase web site and when it is bought from Copay.
Copay calculates the final price after the user introduces how many BTC he wish to buy and coinbase website lets the user introduces the final amount (after coinbase fees  and etc)
Copay should request the total amount and then in the next screen show the fees that let to this final amount introduces by the user

**Comment by cmgustavo** [comment]
The main difference is that Copay needs to send the bitcoin back to your wallet. So, this additional operation (send the purchased bitcoin from Coinbase to Copay) requires a transaction fee, that is different that Coinbase fee to purchase (maybe it 's a bank fee, not sure). In order to send your bitcoin back, after the purchase, Copay creates a new transaction (with the previous charged transaction fee), to send bitcoin from Coinbase to your Copay wallet.

**Comment by cmgustavo** [comment]
Some time ago, this transaction fee was paid by Coinbase. Make a transaction from Coinbase to Copay using our integration was free for fee. Today the history is a little different :(

**Comment by miguelmorales85** [comment]
Then I'll guess there is not a certain added value to use the app to buy bitcoin from Coinbase. I rather Buy directly from the website and then gather some BTC and just move one time the BTC's to Copay (or another wallet)
Thanks for the answer.

**Comment by miguelmorales85** [comment]
Hi @cmgustavo , turns out today I checked the buy order I made form the app and they have Error Status. The message shown is "Cannot send this amount without going over application limit (limit 1000.00 USD, 1000.00 USD remaining)". It was a 2000.00 USD buy request.

Why would the app let me buy an amount bigger than the allowed? This is clearly a bug.

I feel cheated because I had to pay more fiat for the transfer to be made directly to my wallet and now it is in Coinbase so I have to pay again to get my BTC out of there.

**Comment by cmgustavo** [comment]
Hi @miguelmorales85 , I 'm sorry about that. Coinbase limit us to 1,000 USD per day to send money from the user's account. We can't change it because it 's a limit that Coinbase applies for all third part applications.

Just to understand a little more about this bug. Your initial purchase from Copay (2000USD) has been completed on Coinbase.com? I mean, do you have your BTC on Coinbase.com?

Thanks for report.

**Comment by miguelmorales85** [comment]
Yes, I have them in my coinbase account, there is no problem with that. I think you must define a top in the app to avoid users trying to buy more than Coinbase's limit for Copay
